### PR TITLE
Update translation for week of 9/14

### PR DESCRIPTION
**Pull request by chung-lai**
Update translation

### How was this tested?
- [ ] I tested this manually
- [ ] I added automated tests
- [ ] Existing automated tests are enough to cover my changes

### CODE REVIEW

**Review by darinwf — APPROVED** [review]

File: Client/fr.lproj/Localizable.strings [hunk 1]
```diff
@@ -296,7 +269,7 @@
 "arrows" = "flèches";
 
 /* No comment provided by engineer. */
-"Auto Archive Tabs" = "Auto Archive Tabs";
+"Auto Archive Tabs" = "Archivage automatique des onglets";
```
Comment (EdwardJTL) [inline Client/fr.lproj/Localizable.strings:272]: this is still wrong :( did we forget to provide a comment?

Comment (EdwardJTL) [reply Client/fr.lproj/Localizable.strings:272]: its probably okay thought... we are moving away from this label soon anyways.

Comment (chung-lai) [reply Client/fr.lproj/Localizable.strings:272]: lol, oh, we should have changed the settings to use new strings but looks like it's still being used in the new UX `HistoryAndArchivedTabsPanelView`. Think we should change that to `Archived Tabs` or whatever we use in the archived tab settings now. @evancrow

Comment (evancrow) [reply Client/fr.lproj/Localizable.strings:272]: Will do!

**Pull request merged**